clients/python/client/osparc/models.py

```diff
@@ -0,0 +1,46 @@
+
+import warnings
+from ._warnings_and_errors import VisibleDeprecationWarning
+
+warn_msg:str = "osparc.models will be deprecated soon. Instead functionality within this module"
```

Comment:
MINOR: user `Final` and no need to "+=" you can just add multiple lines as
```python
warning_msg: Final[str] = (
 "osparc.models is deprecated. Please use ... "
 "should be imported directly from osparc. I.e. please do 'from osparc import <fcn>' instead of 'from osparc.models import <fcn>'"
)
```

Reply:
done